TS04-Plotting-Time-Series-R.Rmd

```diff
@@ -23,8 +23,7 @@ comments: false
 
 
 ##About
-This activity will walk you through the fundamentals of data manipulation and 
-basic plotting.
+This lesson will teach individuals how to conduct basic data manipulation and create plots of time series data using ggplot2.
```

Comment:
i do'nt like this language. Just say, in this lesson we will... then describe what the lesson covers. teaching participants sounds heavy handed. we want community.